addon/components/tether-tooltip-and-popover.js

```diff
@@ -454,10 +454,13 @@ export default EmberTetherComponent.extend({
 
   stopTether() {
     run.schedule('afterRender', () => {
-      // can't depend on `_tether.enabled` because it's not an
-      // Ember property (so won't trigger cp update when changed)
-      this.set('_isTetherEnabled', false);
-      this.get('_tether').disable();
+      if (!this.isDestroyed) {
```

Comment:
Should we check for `this.isDestroying` instead? I say that because `isDestroying` is set to `true` even earlier than `this.isDestroyed` so it may be more robust.

Also, are you not using `get` because there is no need for a destroyed component? I ask just because I'm not sure myself.